added recipe to use appsettings on server side.

RicoSaupe opened the pull request:
#154 

@isaacabraham @martinbryant 
Please review and let me know if something is missing.

isaacabraham commented:
@RicoSaupe I'm not convinced that any of the packages and middleware is required. `IConfiguration` is available directly in ASP .NET and should be immediately accessible. You can get to `IConfiguration` and then use indexer syntax directly.

RicoSaupe commented:
you are correct. I started out differently and later decided to do less. I removed dependencies and moved my page to the patterns section you created. Please check again
